tests/s25Main/integration/testCheats.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2024 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "Cheats.h"
+#include "GamePlayer.h"
+#include "network/GameClient.h"
+#include "worldFixtures/CreateEmptyWorld.h"
+#include "worldFixtures/WorldFixture.h"
+
+namespace {
+template<unsigned T_numPlayers>
+struct CheatWorldFixture : WorldFixture<CreateEmptyWorld, T_numPlayers>
+{
+    Cheats& cheats = world.GetCheats();
```

Comment:
This needs to be done in the ctor:
>/source/tests/s25Main/integration/testCheats.cpp:15:22: error: 'world' was not declared in this scope; did you mean 'World'?

Reply:
Actually moving this to the ctor does not make a difference. But on gcc/clang `world` needs to be qualified either through `WorldFixture<CreateEmptyWorld, T_numPlayers>::world` or `this->world`.